### PR TITLE
WA-85: Splitup transfer flow

### PR DESCRIPTION
**Pull request by rmeissner**
Closes WA-85.

Changes proposed in this pull request:
- Transfer flow is split up into create and review
- Added asset transfer review screen

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/java/pm/gnosis/heimdall/ui/transactions/ViewTransactionActivity.kt [hunk 1]
```diff
@@ -24,10 +40,103 @@ class ViewTransactionActivity : BaseTransactionActivity() {
...
+
+    override fun onStart() {
+        super.onStart()
+        setUnknownTransactionInfo()
+        disposables += layout_view_transaction_submit_button.clicks()
+                .flatMap {
+                    currentInfo?.let { (safeAddress, transaction) ->
+                        safeAddress?.let {
+                            viewModel.submitTransaction(safeAddress, transaction)
+                                    .subscribeOn(AndroidSchedulers.mainThread())
```
Comment (fmrsabino) [inline app/src/main/java/pm/gnosis/heimdall/ui/transactions/ViewTransactionActivity.kt]: needed?

File: app/src/main/res/layout/layout_review_asset_transfer.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:gravity="center"
+    android:padding="16dp">
+
+    <TextView
+        style="@style/Content"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="This transaction will transfer" />
```
Comment (fmrsabino) [inline app/src/main/res/layout/layout_review_asset_transfer.xml]: string resource

File: app/src/main/res/layout/layout_review_asset_transfer.xml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        android:padding="8dp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        tools:text="3450 GNO" />
+
+    <TextView
+        style="@style/Content"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="from your Safe" />
```
Comment (fmrsabino) [inline app/src/main/res/layout/layout_review_asset_transfer.xml]: string resource

File: app/src/main/res/layout/layout_review_asset_transfer.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        android:padding="8dp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        tools:text="Gnosis Safe (0x001234)"/>
+
+    <TextView
+        style="@style/Content"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="to" />
```
Comment (fmrsabino) [inline app/src/main/res/layout/layout_review_asset_transfer.xml]: string resource

File: app/src/main/res/layout/layout_review_asset_transfer.xml (hunk 2 above)
Comment (rmeissner) [reply app/src/main/res/layout/layout_review_asset_transfer.xml]: the review layout will be adjusted once we have the design, I would postpone it until then. ok?

File: app/src/main/java/pm/gnosis/heimdall/ui/transactions/ViewTransactionActivity.kt (hunk 1 above)
Comment (rmeissner) [reply app/src/main/java/pm/gnosis/heimdall/ui/transactions/ViewTransactionActivity.kt]: https://gnosisltd.atlassian.net/browse/WA-96

**Pull request merged**